soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -521,47 +521,47 @@ double Offense::distance_from_their_robots(rj_geometry::Point tail, rj_geometry:
     return min_angle;
 }
 
-bool Offense::can_steal_ball() const {
-    // Ball in red zone or not
-    if (ball_in_red()) {
-        return false;
-    }
-    // Ball location
-    rj_geometry::Point ball_position = this->last_world_state_->ball.position;
-
-    // Our robot is closest robot to ball
-    bool closest = true;
-
-    auto current_pos = last_world_state_->get_robot(true, robot_id_).pose.position();
-
-    auto our_dist = (current_pos - ball_position).mag();
-    for (auto enemy : this->last_world_state_->their_robots) {
-        auto dist = (enemy.pose.position() - ball_position).mag();
-        if (dist < our_dist) {
-            closest = false;
-            break;
-        }
-    }
-
-    if (!closest) {
-        return closest;
-    }
-
-    for (auto pal : this->last_world_state_->our_robots) {
-        // if (pal.robot_id_ == robot_id_) {
-        // continue;
-        // }
-        auto dist = (pal.pose.position() - ball_position).mag();
-        if (dist < our_dist) {
-            closest = false;
-            break;
-        }
-    }
-
-    return closest;
-
-    // return distance_to_ball() < kStealBallRadius;
-}
+// bool Offense::can_steal_ball() const {
```

Comment:
not a fan of commenting-out methods. either we need them or we delete them, IMO. if we temporarily don't need them, then at least leave a comment explaining. but honestly no harm in leaving them there, in that case.